### DISCUSSION TITLE
Raising an Issue

### DISCUSSION THREAD

**Discussion by goobric** [EddieHubCommunity/LinkFree#9693] [Ideas]
On the 'New Issues' page these are the following categories:
- Bug
- General Feature Request
- Testimonial
- Report a security vulnerability
- Question

There isn't a section for 'Documentation'
The following are normally understood to be:
Bug -> normally implies there is an issue with the 'code'
General Feature Request -> add to the product
Testimonial -> give a positive review
Report vulnerability -> some security issue; usually linked to a 'data breach'
Question -> Ask something.

I would like to submit an issue regarding the documentation; which of the above categories should I select; if:
- the product works
- I don't want to add a feature
- not writing a testimonial
- no security issue
- not a general question about the product
![BD_Issue_Page](https://github.com/EddieHubCommunity/BioDrop/assets/23141638/25201b26-7be6-4905-bd27-703ceaaf2ef7)

**Comment by eddiejaoude** [comment]
Thank you for your suggestion. We treat all Giles in the repo, like documentation and tests just like code. They can have features as well as bugs. I don't think a new template is required.
